delivery.yaml

```diff
@@ -74,12 +70,6 @@ pipeline:
         cdp-promote-image "${MULTIARCH_IMAGE}"
         echo "Created multi-arch docker image container-registry.zalando.net/teapot/skipper:${RELEASE_VERSION}"
 
-        echo "Creating docker image registry.opensource.zalan.do/teapot/skipper-arm64:${RELEASE_VERSION}"
-        make docker.build.arm64 && git status && git diff && make docker.push.arm64
-
-        echo "Creating docker image registry.opensource.zalan.do/teapot/skipper-armv7:${RELEASE_VERSION}"
-        make docker.build.armv7 && git status && git diff && make docker.push.armv7
-
```

Comment:
Should we clean up Makefiles and Dockerfiles?

Reply:
I think multiarch is right now only arm64 and am64, so cleaning up armv7 (RPIs <v4) would not make sense.
I think it's fine to have these targets, but maybe we can also re-think it